docs/09.EXT-REFERENCE-ARG.md

```diff
@@ -461,7 +462,7 @@ jerryx_arg_boolean (bool *dest,
 **Summary**
 
 Create a validation/transformation step (`jerryx_arg_t`) that expects to
-consume one `string` JS argument and stores it into a C `char` array.
+consume one `string` JS argument and stores it into a cesu8 C `char` array.
```

Comment:
"CESU-8"? I think that's how one would normally write it (outside of the context of code). For example https://en.wikipedia.org/wiki/CESU-8